### PR TITLE
T&C screen styles | Check if if the user has accepted T&C

### PR DESCRIPTION
**Pull request by aarroyo06**
@malvarezhcr, please review.
- feature/project-setup/MCBA-34 – Adding Terms and Conditions style page | Handling logic to get terms & conditions copy from DB
- feature/project-setup/MCBA-35 – Adding logic to check if the user has accepted T&C policies in a previous time.

### CODE REVIEW

File: app/app.ts [hunk 1]
```diff
@@ -13,12 +13,18 @@ import { DBService,
          AuthService,
          RequestService }       from './global/services/';
 
-import { LoginPage, TabsPage }  from './pages/';
+import { TermsConditionsPage,
```
Comment (malvarezhcr) [inline app/app.ts]: @aarroyo06  please order alphabetically

File: app/pages/initialsetup/termsconditions/terms-conditions.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<ion-content class="terms-conditions-page">
+
+  <div class="app-logo">
```
Comment (malvarezhcr) [inline app/pages/initialsetup/termsconditions/terms-conditions.html]: @aarroyo06  please user text-center attribute

File: app/pages/initialsetup/termsconditions/terms-conditions.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.terms-conditions-page {
+	.app-logo {
+		text-align: center;
```
Comment (malvarezhcr) [inline app/pages/initialsetup/termsconditions/terms-conditions.scss]: @aarroyo06 please remove text-align, width, display properties, I think they are not needed here

File: app/pages/initialsetup/termsconditions/terms-conditions.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<ion-content class="terms-conditions-page">
```
Comment (malvarezhcr) [inline app/pages/initialsetup/termsconditions/terms-conditions.html]: @aarroyo06 please remove this class "terms-conditions-page" It's being set automatically to the ion-page

File: app/pages/initialsetup/termsconditions/terms-conditions.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+    span {
+      display: table;
+      margin: 0 auto;
+      font-weight: bold;
+    }
+  }
+
+  .call-to-action {
+    padding: 31px 51px 55px
```
Comment (malvarezhcr) [inline app/pages/initialsetup/termsconditions/terms-conditions.scss]: @aarroyo06 for sizes bigger than 10px please use rems instead. 1rem is equal to 10px

File: app/theme/app.core.scss [hunk 6]
```diff
@@ -31,6 +31,7 @@
 @import "../pages/initialsetup/forgot-password/forgot-password";
 @import "../pages/home/home";
 @import "../pages/tabs/tabs";
+@import "../pages/initialsetup/termsconditions/terms-conditions";
```
Comment (malvarezhcr) [inline app/theme/app.core.scss]: @aarroyo06 please order alphabetically

File: app/pages/initialsetup/termsconditions/terms-conditions.scss [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  .content {
+    margin: 0 22px;
+    border-radius: 4px;
+    border: 2px solid $color-dark-grey;
+    padding: 12px 6px 10px;
+    overflow: auto;
+    height: 58vh;
+    font-size: 15px;
+
+    span {
```
Comment (malvarezhcr) [inline app/pages/initialsetup/termsconditions/terms-conditions.scss]: @aarroyo06 please use a headline tag (h1,h2,3) and text-center attribute so you need to align it center with margins.

File: app/pages/initialsetup/termsconditions/terms-conditions.ts [hunk 8]
```diff
@@ -0,0 +1,38 @@
+import { Component }     from '@angular/core';
+import { NavController } from 'ionic-angular';
+
+import { LoginPage }  from '../login/login';
```
Comment (malvarezhcr) [inline app/pages/initialsetup/termsconditions/terms-conditions.ts]: @aarroyo06 please order alphabetically and use relative path to aim index.ts

File: app/constants/keys.constants.ts [hunk 9]
```diff
@@ -0,0 +1,4 @@
+export const MCBA_SESSION_TIMESTAMP = 'session_timestamp';
```
Comment (malvarezhcr) [inline app/constants/keys.constants.ts]: hey @aarroyo06 please rename this file to db-keys.constants.ts

File: app/app.ts (hunk 1 above)
Comment (aarroyo06) [inline app/app.ts]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/pages/initialsetup/termsconditions/terms-conditions.html (hunk 2 above)
Comment (aarroyo06) [inline app/pages/initialsetup/termsconditions/terms-conditions.html]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/pages/initialsetup/termsconditions/terms-conditions.scss (hunk 3 above)
Comment (aarroyo06) [inline app/pages/initialsetup/termsconditions/terms-conditions.scss]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/pages/initialsetup/termsconditions/terms-conditions.html (hunk 4 above)
Comment (aarroyo06) [inline app/pages/initialsetup/termsconditions/terms-conditions.html]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/pages/initialsetup/termsconditions/terms-conditions.scss (hunk 5 above)
Comment (aarroyo06) [inline app/pages/initialsetup/termsconditions/terms-conditions.scss]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/theme/app.core.scss (hunk 6 above)
Comment (aarroyo06) [inline app/theme/app.core.scss]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/pages/initialsetup/termsconditions/terms-conditions.scss (hunk 7 above)
Comment (aarroyo06) [inline app/pages/initialsetup/termsconditions/terms-conditions.scss]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/pages/initialsetup/termsconditions/terms-conditions.ts (hunk 8 above)
Comment (aarroyo06) [inline app/pages/initialsetup/termsconditions/terms-conditions.ts]: @malvarezhcr updated on GIT commit: efe8bf4

File: app/constants/keys.constants.ts (hunk 9 above)
Comment (aarroyo06) [inline app/constants/keys.constants.ts]: @malvarezhcr updated on GIT commit: efe8bf4

**Pull request merged**